testdata/p4_16_samples_outputs/ipv4-actions_ubpf.c

```diff
@@ -0,0 +1,358 @@
+/* Automatically generated by p4c-ubpf from /home/mateusz/p4c/testdata/p4_16_samples/ipv4-actions_ubpf.p4 on Mon Jan 20 15:27:07 2020
```

Comment:
Do you want to save these files in the repository?
Are you comparing with them in some tool? 
The ebpf back-end does not save the C files - I was afraid that they would change too often with compiler changes to make for good tests, but perhaps that's wrong.

Reply:
You're it is not the best way, but I think it is the only reasonable (at least for now) way to test if the C files generated by `p4c-ubpf` are correct. We compare them at very basic level, using a minimalistic Python code.

We are aware that it could require changes to these file once we will add some new functionality to `p4c-ubpf`. I would keep these files for now, until we will integrate P4rt-OVS (or some other uBPF verifier) with `make check-ubpf`.

Reply:
We've removed these files and replaced them by userspace-only tests (based on those implemented for `ebpf`)